dcpy/models/product/metadata.py

```diff
@@ -123,6 +124,7 @@ class OrgMetadata(SortedSerializedBase, extra="forbid"):
     template_vars: dict = Field(default_factory=dict)
     metadata: OrgMetadataFile
     column_defaults: dict[tuple[str, COLUMN_TYPES], DatasetColumn]
+    data_dictionary: DataDictionary = DataDictionary()
```

Comment:
A bit pedantic, but I think I'd prefer something like `DataDictionarySpec` just to clarify that this is a definition of the format of the data dictionary rather than an instance of one